.github/workflows/check-new-library-versions.yml

```diff
@@ -0,0 +1,142 @@
+name: "Check new library versions"
+
+on:
+  schedule:
+    - cron: "0 0 8 * *"
```

Comment:
Is this due to the release schedule? 

Would it be possible to run more often? That way we would have time to react before the release.

Reply:
The idea is to have library updates exactly in the middle between two releases. This way, we are giving some time after the release to aggregate more new library versions but still to have one week to react before the next release.

In my opinion, having more frequent scans (lets say every week) won't help us much, because we won't have time to react in those weeks when we have a scheduled release.

Reply:
Write this down as a comment.